Documentation/cmdref/cilium-agent.md

```diff
@@ -311,7 +311,6 @@ cilium-agent [flags]
       --route-metric int                                          Overwrite the metric used by cilium when adding routes to its 'cilium_host' device
       --routing-mode string                                       Routing mode ("native" or "tunnel") (default "tunnel")
       --service-no-backend-response string                        Response to traffic for a service without backends (default "reject")
-      --sidecar-istio-proxy-image string                          Regular expression matching compatible Istio sidecar istio-proxy container image names (default "cilium/istio_proxy")
```

Comment:
@sayboras  drive-by note (knowing that it's still in draft): the property is also configured in the [Cilium Configmap](https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/templates/cilium-configmap.yaml#L430-L432) and the [Helm Chart values](https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/values.yaml.tmpl#L1779-L1781).

Reply:
ah good points, I actually missed the above, thanks a lot.